corehq/apps/export/views.py

```diff
@@ -2283,6 +2286,56 @@ def get_filters(self, filter_form_data, mobile_user_and_group_slugs):
         return form_filters
 
 
+class DownloadNewSmsExportView(GenericDownloadNewExportMixin, BaseDownloadExportView):
```

Comment:
i would've expected to have seen more views like the edit view and create view. are those not for this PR?